.github/workflows/rust.yml

```diff
@@ -118,8 +118,8 @@ jobs:
         uses: actions/cache@v1
         with:
           path: rust/target
-          key: windows-rust-${{ hashFiles('rust/**.rs') }}
-          restore-keys: windows-rust-
+          key: windows-rust-v1-${{ hashFiles('rust/**/**.rs', 'rust/**/Cargo.toml') }}
```

Comment:
Is `v1` used here for invalidating the previous cache?

Reply:
yes, purely for that. Maybe there is a way to address this in another way, I was unable to make it work.